databricks/koalas/frame.py

```diff
@@ -492,6 +492,104 @@ def to_dict(self, orient='dict', into=dict):
         return validate_arguments_and_invoke_function(
             kdf.to_pandas(), self.to_dict, pd.DataFrame.to_dict, args)
 
+    def to_csv(self, path_or_buf=None, sep=",", na_rep='', float_format=None,
+               columns=None, header=True, index=True, index_label=None,
+               mode='w', encoding=None, compression='infer', quoting=None,
+               quotechar='"', line_terminator=None, chunksize=None,
+               tupleize_cols=None, date_format=None, doublequote=True,
+               escapechar=None, decimal='.'):
+        r"""
+        Write object to a comma-separated values (csv) file.
+            The order of arguments for Series was changed.
```

Comment:
also need a note warning saying this could blow up. you can copy the warning from to_pandas

Reply:
Agree !